src/sessions/softphone-session-handler.ts

```diff
@@ -392,6 +392,9 @@ export class SoftphoneSessionHandler extends BaseSessionHandler {
     callState: ICallStateFromParticipant
   ): void {
     if (callState.errorInfo) {
+      if (callState.errorInfo.code === 'error.ininedgecontrol.session.inactive' && callState.errorInfo.message.includes('is inactive for')) {
```

Comment:
I'll be honest, I'm not entirely sure if this is the best check... I'm not 100% sure if just checking the code is sufficient so I added a check for the message too but I'm not sure if _that_ is the best check either. I imagine we would get the translated version in the case of using a different language?

Reply:
I don't think any of our error messaging is translated at this level but we shouldn't be checking for error text and should just check for the error code.

Reply:
My concern was just checking the error code would result in us swallowing non-benign errors that we actually wanted to show.  Are we fine to just check the code without other errors being affected?  I agree with both of you guys though.

Reply:
Probably worth asking the Edge guys.